src/components/AlertModal/alertModal.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from "react";
+import Snackbar from "@mui/material/Snackbar";
+import { AlertModalProps } from "./types";
+import { AlertContainer } from "./styled";
+
+const AlertModal: FC<AlertModalProps> = ({
+  handleCloseAlertModal,
+  errorMessage = "Something wrong happened",
+  vertical = "top",
+  horizontal = "right",
```

Comment:
Since all of our alerts are going to be top right we can remove these parameters and pass them directly to Snackbar, what do you think?